clients/client-python/gravitino/filesystem/gvfs.py

```diff
@@ -567,6 +569,9 @@ def _convert_actual_path(
             or storage_location.startswith(f"{StorageType.S3A.value}://")
         ):
             actual_prefix = infer_storage_options(storage_location)["path"]
+        elif storage_location.startswith(f"{StorageType.OSS.value}:/"):
+            ops = infer_storage_options(storage_location)
+            actual_prefix = ops["host"] + ops["path"]
```

Comment:
it's better to check the `ops` whether have `host` and `path` attributes.

Reply:
OK